.github/workflows/ci.yml

```diff
@@ -27,19 +27,42 @@ jobs:
           - 2.2
           - 2.1.9
           - ruby-head
+        env:
+          -
+            DIFF_LCS_VERSION: "> 1.4.3"
+        include:
+          - ruby: jruby-9.2.13.0
+            env:
+              JRUBY_OPTS: "--dev"
+          - ruby: jruby-9.1.17.0
+            bundler: 1
+            env:
+              JRUBY_OPTS: "--dev"
+          - ruby: 2.7
+            name_extra: "with diff-lcs 1.3"
+            env:
+              DIFF_LCS_VERSION: "~> 1.3.0"
+          - ruby: 2.7
+            name_extra: "with diff-lcs 1.4.3"
+            env:
+              DIFF_LCS_VERSION: "1.4.3"
       fail-fast: false
-    continue-on-error: ${{ matrix.ruby == 'jruby-9.2.13.0' || endsWith(matrix.ruby, 'head') }}
+    continue-on-error: ${{ matrix.allow_failure || endsWith(matrix.ruby, 'head') }}
```

Comment:
Do you need the first allow_failure? I do not see it in the matrix.

Reply:
Its there to allow failures if required.